### PR TITLE
fix(styles): align support styles with the rest of fxa

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

**Comment by johngruen** [comment]
Fixes #2936

**Comment by johngruen** [comment]
@chenba wanted to make sure the styles in the support page better matched settings in general

File: packages/fxa-content-server/app/scripts/templates/support.mustache [hunk 1]
```diff
@@ -25,17 +25,10 @@
...
-            <header class="settings-unit-summary">
-              <h2 class="settings-unit-title">Subscriptions</h2>
-            </header>
-          </div>
-        </div>
         <div class="settings-unit">
           <div class="support-form">
             <header>
-              <h3>{{#t}}Contact Us{{/t}}</h3>
+              <h2>{{#t}}Contact Us{{/t}}</h2>
```
Comment (LZoog) [inline packages/fxa-content-server/app/scripts/templates/support.mustache]: I think this should remain an `h3` as it makes sense for it to fall under "Subscriptions". (It looks like we have an empty `h2` rendered if there's no display name, I may not have latest though)

image

File: packages/fxa-content-server/app/styles/modules/_chosen.scss [hunk 2]
```diff
@@ -71,9 +71,9 @@
 
   .chosen-single {
     div {
-      border-color: #686869 transparent transparent transparent;
+      border-color:  $grey-30 transparent transparent transparent;
```
Comment (LZoog) [inline packages/fxa-content-server/app/styles/modules/_chosen.scss]: 🙌 for variable usage

File: packages/fxa-content-server/app/scripts/templates/support.mustache (hunk 1 above)
Comment (johngruen) [reply packages/fxa-content-server/app/scripts/templates/support.mustache]: sure!

**Comment by johngruen** [comment]
Made @LZoog's requested change and merged

**Pull request merged**